tests/test_schema.py

```diff
@@ -45,3 +45,80 @@ def test_relative_path_not_match_schema(data, schema):
     data["rhui_alias"][0]["dest"] = "dest-path"
     with pytest.raises(ValidationError):
         jsonschema.validate(data, schema)
+
+
+def test_empty_list(data, schema):
+    """Verify that YAML returns empty list."""
```

Comment:
This doesn't seem necessary. We shouldn't need to test the functionality of a third-party library.